feat/new product overview pages (#646)
hirosystems/docs: docs/stacks.js/index.mdx

    <div class="subSectionTitle"><h3>Sign messages</h3><span>→</span></div>
    <p>Prompt users to sign messages and prove they control an address or authorize an in-app action.</p>
  </a>
  <a href="/stacks.js/learn-the-basics">
    <div class="subSectionTitle"><h3>Understand the basics</h3><span>→</span></div>
    <p>Discover the basics of accounts, networks, and transactions.</p>
  </a>
</div>

<div class="subSections my-8 w-[50%]">
  <a href="/stacks.js/guides/deep-dive-post-conditions">
    <div class="subSectionTitle"><h3>Constructing Post-Conditions</h3><span>→</span></div>
    <p>Build post conditions into your smart contracts and safe-guard your users.</p>
  </a>
</div>

### Related Tools

**[Clarinet](../clarinet/index.mdx)**: Build your app in the go-to development environment on Stacks.

**[Hiro Platform](../platform/index.mdx)**: Build your app in the cloud in a hosted environment.

**[Stacks Blockchain API](../stacks-blockchain-api/index.mdx)**: Fetch data and broadcast transactions with the Stacks Blockchain API.

**[Stacks Explorer](https://explorer.hiro.so)**: Explore and verify on-chain your user’s activity on Stacks.

---

> Need help building with Stacks.js? Reach out to us on the `#stacks-js` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with Stacks.js? Reach out to us on the `#stacks-js` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
